### PR TITLE
fix: update swagger ui annotations for info and status params

### PR DESCRIPTION
**Pull request by edalex-yinzi**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] tests are included
- [x] screenshots are included showing significant UI changes
- [ ] documentation is changed or added

##### Description of change
Update annotations for api `search`.
Including status and info params.
Before:
image

After:
image

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Review by edalex-ian — COMMENTED** [review]
Hmmm, we've lost info here. Now people can't see what values are available. :thinking:

**Comment by edalex-yinzi** [comment]
How about this:
image

**Comment by edalex-ian** [comment]
hmmm, please see the discussion in the thread in slack I mentioned you in.

**Comment by edalex-yinzi** [comment]
Tried to set serialization param but not work.

File: Source/Plugins/Core/com.equella.core/src/com/tle/web/api/search/EquellaSearchResource.java [hunk 1]
```diff
@@ -127,11 +128,11 @@
...
       @ApiParam(
-              value = "Filter by item status",
-              required = false,
-              allowableValues =
-                  "DRAFT,LIVE,REJECTED,MODERATING,ARCHIVED,SUSPENDED,DELETED,REVIEW,PERSONAL",
-              allowMultiple = true)
+              value =
+                  "Filter by item status"
+                      + "\nAvailable values: "
+                      + "DRAFT,LIVE,REJECTED,MODERATING,ARCHIVED,SUSPENDED,DELETED,REVIEW,PERSONAL",
```
Comment (edalex-ian) [inline Source/Plugins/Core/com.equella.core/src/com/tle/web/api/search/EquellaSearchResource.java:134]: Seeing this is now doco, let's make sure we include spaces:

```suggestion
                      + "DRAFT, LIVE, REJECTED, MODERATING, ARCHIVED, SUSPENDED, DELETED, REVIEW, PERSONAL",
```

But also, I note above (for `info`) you've used the enum directly. Is that an option here? But the other thing is, in one you're saying the values should be lower case and the other upper case - be good to be consistent.

**Review by edalex-ian — CHANGES_REQUESTED** [review]
Okay, so as per discussions I agree let's just stick with a raw string for now.

BUT, we want to tidy up the formatting of the doco strings.

File: Source/Plugins/Core/com.equella.core/src/com/tle/web/api/search/EquellaSearchResource.java (hunk 1 above)
Comment (edalex-yinzi) [reply Source/Plugins/Core/com.equella.core/src/com/tle/web/api/search/EquellaSearchResource.java]: No, but I can add one.

**Comment by edalex-yinzi** [comment]
Format string ALL_ALLOWABLE_INFOS and add a string ALL_ALLOWABLE_STATUSES in IteamResource.

**Comment by edalex-yinzi** [comment]
image

**Review by edalex-ian — APPROVED** [review]
Thanks @edalex-yinzi :+1:

**Pull request merged**